<launch>
  <arg name="robot_name" value="DB1"/>
 <include file="$(find cordial_face)/launch/face.launch">
      <arg name="name" value="$(arg robot_name)"/>
  </include>
  <node name="setup_http_servers" pkg="cordial_face" type="setup_http_servers.sh" output ="screen" args="8081 8082"/>

<node name="run_pi_browser" pkg="cordial_face" type="client_to_start_face_on_pi.py" output ="screen" />

</launch>
